contracts/WooracleV2.sol

```diff
@@ -185,7 +185,7 @@ contract WooracleV2 is InitializableOwnable, IWooracleV2 {
         timestamp = block.timestamp;
     }
 
-    function price(address base) external view override returns (uint256 price, uint256 priceTimestamp) {
+    function price(address base) external view override returns (uint256 _price, uint256 priceTimestamp) {
```

Comment:
In general, '_price' is used for params or local variable. Better to rename it as 'priceOut'.